scripts/devops_tasks/find_change_log.ps1

```diff
@@ -0,0 +1,105 @@
+﻿
+param (
+  $workingDir,
+  $version
+)
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?))"
+
+
+function ExtractReleaseNotes($changeLogLocation)
```

Comment:
I'm fine with duplicating this in the python repo for GA but please file an issue to share this logic with the release process as I don't want us to maintain it across multiple locations.

Reply:
I have filed issue #8244 and self assigned.